### DISCUSSION TITLE
How can I validate the private key (or ED25519 private key) get from mobile when login success on Backend-side?

### DISCUSSION THREAD

**Discussion by huynhvansang122** [Web3Auth/Web3Auth#714] [Help]
I have received user info when login success to web3auth. (private key, ed25519 private key, username, email, avatar,...).
How can I validate this private key in the backend? I use Google social login and email login.

**Comment by shahbaz17** [comment]
Hey @huynhvansang122 

May I know what use case you're looking and why you want to validate user's private key?

Also, to do so, you would need to pass the private key to the backend. Which might not be a good way to validate. If your purpose is to validate a user. Then please check out https://web3auth.io/docs/server-side-verification/social-login-users#verifying-idtoken-in-backend to verify.
